raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+Strategic environmental assessment for mineral, petroleum, hydro-electric power and major water project plans 
+(105) [1] Where a mineral or petroleum resource is identified and before specific details are planned or a hydro-electric power station is planned or a major water project is planned, the Ministry responsible for mining, energy or water shall carry out a Strategic Environmental Assessment. [2] The Strategic Environmental Assessment provided for under subsection [1], shall assess the area marked for development including the following: [a] baseline environmental conditions and status of natural resources; [b] identification of ecologically sensitive and protected areas; [c] identification and description of communities around the area; [d] existing socio-economic conditions; [e] existing economic activities and infrastructure; [f] proposed developments, including long term scenarios and the cumulative development of a number of different mine or oil and gas site or hydro-electric power stations; [g] infrastructure and resources required to service these development; [h] potential environmental and social impacts of mining or petroleum development or hydro-electric power or any major water projects; and [i] recommendations for land reclamation and limitations on development in different areas. [3] The Minister shall direct the Director of Environment to review the strategic environmental assessment statement and prepare a report on adequacy or inadequacy of the statement and areas which need to be improved and make recommendations to the Minister accordingly. [4] The Director of Environment shall submit a review report to the Minister for approval. [5]. On receipt of recommendations, the Minister shall approve the report and make recommendations to the Ministry responsible for mining, energy or water regarding the best way environment can be preserved within the context of the project that is to be undertaken. [6] The Ministry responsible for mining, energy or water projects shall be required to comply with recommendations made by the Minister. 
```

Comment:
Please replace:
* `mine or oil and gas site` with `mine or oil and gas sites`; and
* `these development` with `these developments`.